api/v1/db/json/notes-dao.js

```diff
@@ -73,6 +73,35 @@ const getNotes = query => new Promise((resolve, reject) => {
   }
 });
 
+/**
+ * @summary Fetch a note from the database by its noteID
+ * @function
+ * @param noteID
+ * @returns {Object} The raw note from the DB
+ */
+const fetchNote = (noteID) => {
+  try {
+    const studentID = noteID.split('-')[0];
+    const studentDirPath = `${dbDirectoryPath}/${studentID}`;
+    return JSON.parse(fs.readFileSync(`${studentDirPath}/${noteID}.json`, 'utf8'));
+  } catch (err) {
+    return null;
+  }
+};
+
+/**
+ * @summary Write newContents to the note with id noteID
+ * @function
+ * @param noteID
+ * @param newContents
+ * @param options Additional options for writeFileSync
+ */
+const writeNote = (noteID, newContents, options = {}) => {
```

Comment:
Will the option be different in any case? It seems like when calling `writeNote` function the option will only be `{ flag: 'wx' }`.

Reply:
Right, I suppose the `options` parameter could be changed to a simple boolean that would determine if `{ flag: 'wx' }` is used since no other options are needed in this api.